pysat/_instrument.py

```diff
@@ -39,7 +40,8 @@ class Instrument(object):
     inst_id : string, optional
         identity within constellation
     clean_level : {'clean','dusty','dirty','none'}, optional
```

Comment:
```suggestion
    clean_level : str or NoneType
```

Add string options to the description

Also, `, optional` isn't a typical feature of the docstrings.

Reply:
Accepted. That docstring has probably been around for awhile. Though we aren't really using it, the optional notation is part of the numpydoc standard.

```
If it is not necessary to specify a keyword argument, use optional:

x : int, optional
```